modules/google-cloud-cpp/2.16.0/patches/googleapis.patch

```diff
@@ -0,0 +1,47 @@
+diff --git a/bazel/extensions.bzl b/bazel/extensions.bzl
+new file mode 100644
+index 0000000000..2002631bc7
+--- /dev/null
++++ b/bazel/extensions.bzl
+@@ -0,0 +1,26 @@
++load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
++load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
++
++def non_module_deps_impl(_ctx):
++    # Load the googleapis dependency.
++    maybe(
++        http_archive,
++        name = "com_google_googleapis",
++        urls = [
++            "https://storage.googleapis.com/cloud-cpp-community-archive/com_google_googleapis/d482d7e5793b7444260526ca813b5d1e8b3be582.tar.gz",
++            "https://github.com/googleapis/googleapis/archive/d482d7e5793b7444260526ca813b5d1e8b3be582.tar.gz",
++        ],
++        sha256 = "60ce1e426265b373b4e2f60241c328865a2061a5304c5e7fb9afb8f17bc68520",
++        strip_prefix = "googleapis-d482d7e5793b7444260526ca813b5d1e8b3be582",
++        build_file = Label("//bazel:googleapis.BUILD"),
++        # Scaffolding for patching googleapis after download. For example:
++        #   patches = ["googleapis.patch"]
++        # NOTE: This should only be used while developing with a new
++        # protobuf message. No changes to `patches` should ever be
++        # committed to the main branch.
++        patch_tool = "patch",
++        patch_args = ["-p1"],
++        patches = [],
++    )
++
++non_module_deps = module_extension(implementation = non_module_deps_impl)
```

Comment:
I did merge a version of googleapis into the BCR, can we use that version or do we need a different one for this? i noticed ODR violations if we have 2

Reply:
is it missing the `googleapis_system_includes` target? seems like that's a bespoke thing for this repo https://github.com/googleapis/google-cloud-cpp/blob/f28d70b78f291e3c2074b25dd16c22fbed79cb97/bazel/googleapis.BUILD#L25C13-L25C39

Maybe we can patch that in somehow instead?